packages/account-web-vault/src/index.ts

```diff
@@ -1,7 +1,9 @@
 import VaultAccount, {hasSession} from "./vault-account"
 import { VaultAccountConfig } from "./interfaces"
+import { WebUser } from "./web-user"
```

Comment:
The interfaces are not exported. This can be annoying from a DX point of view.

I would personally simply do:

```ts
export * from "./web-user"
```

Same for the other files.

To me all Types and Interfaces should be externally accessible.

If there are private methods, they should be either kept private in the scope of their file, or in a dedicated file not re-exported by this root `index.ts` (or any other down the hierarchy)

Reply:
Agreed